azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunnerManager.java

```diff
@@ -625,10 +626,17 @@ private void deleteExecutionDir(final int executionId) {
 
   @Override
   public void handleEvent(final Event event) {
-    if (event.getType() == EventType.FLOW_FINISHED || event.getType() == EventType.FLOW_STARTED) {
-      final FlowRunner flowRunner = (FlowRunner) event.getRunner();
-      final ExecutableFlow flow = flowRunner.getExecutableFlow();
+    // Adding extra logging for debuggability of execution directory cleanup call
+    final FlowRunner flowRunner = (FlowRunner) event.getRunner();
+    final ExecutableFlow flow = flowRunner.getExecutableFlow();
 
+    if (event.getType() != null && flow != null) {
```

Comment:
If Azkaban supports configuring the log level then these messages can be added as logger.debug. They can be turned on as needed by changing the log level to debug.

Reply:
I believe we have other logs which are debug level too. It would cause a see of logs being added. Not terrible but could be avoided.

Reply:
@li-gsalia  & @djaiswal83 It will be lot many logs if we will change it to DEBUG. Is it ok if we keep it at info level?

Reply:
Adding debug logs.